### PR TITLE
User response by id added

### PR DESCRIPTION
**Pull request by sabinkarki**
Added UserController and endpoint userbyid  #7

### CODE REVIEW

File: guru-finder/src/main/java/com/miyoz/gurufinder/api/models/interfaces/impl/UserRequest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package com.miyoz.gurufinder.api.models.interfaces.impl;
+
```
Comment (binaydesar) [inline guru-finder/src/main/java/com/miyoz/gurufinder/api/models/interfaces/impl/UserRequest.java]: please remove spaces

File: guru-finder/src/main/resources/application-local.yml [hunk 2]
```diff
@@ -1,15 +1,19 @@
 server:
-  port: 8082
-  
+  port: 8080
```
Comment (binaydesar) [inline guru-finder/src/main/resources/application-local.yml]: I think port should be 8082 . Please check.

Comment (miyoz) [inline guru-finder/src/main/resources/application-local.yml]: Yes, leave the port 8082 and if you are using 8080 on your local, put that application-local.yml in gitingore for you.

Comment (sabinkarki) [inline guru-finder/src/main/resources/application-local.yml]: Done.  
https://github.com/miyoz/gurufinder/blob/sabinK/guru-finder/src/main/resources/application-local.yml#L2

File: guru-finder/src/main/java/com/miyoz/gurufinder/api/models/interfaces/impl/UserRequest.java (hunk 1 above)
Comment (sabinkarki) [inline guru-finder/src/main/java/com/miyoz/gurufinder/api/models/interfaces/impl/UserRequest.java]: Done. 
https://github.com/miyoz/gurufinder/blob/sabinK/guru-finder/src/main/java/com/miyoz/gurufinder/api/models/interfaces/impl/UserRequest.java#L2

**Comment by miyoz** [comment]
@sabinkarki please close this pull request and make new pull request with issue #20 following pull request standard. Thanks.

**Pull request closed without merging**